cfgov/jinja2/v1/wagtail/molecules/contact-email.html

```diff
@@ -0,0 +1,26 @@
+{# ==========================================================================
+
+   contact_email.render()
```

Comment:
Doesn't look like the `render` function exists anymore. See lack of it on https://github.com/cfpb/cfgov-refresh/pull/1127/files#diff-347ae1fb8d2a9b26ea81e49439e05ee2R3 too.